LondonBritishLibrary/orient/BLorient489.xml

```diff
@@ -240,14 +248,15 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         
                         
                         <bindingDesc>
-                            <binding contemporary="true" xml:id="binding">
+                            <binding contemporary="partly" xml:id="binding">
                                 
-                                <decoNote xml:id="b0">In the original boards. </decoNote>
+                                <decoNote xml:id="b0">Wooden boards with leather quarter cover. The manuscript might have been rebound by the library.
```

Comment:
Above, there is no formula "The manuscript might have been...." whereas some of them - if not all - definitely are. Should this fact be indicated better through "Restoration modern"?

Reply:
"The manuscript has been rebound in the Library" has been suggested by you for BLorient482 above, I used it also for other manuscripts in this PR, with different degrees of certainty. I can add  "Restoration modern" in the relatively certain cases, but I would also include a narrative description, because "Restoration modern" could mean many things.

Reply:
Now is the moment to discuss it, as it has not yet been frequently used :) Do you find the formulation problematic in general or is its use here irregular?